### PR TITLE
UI/UX improvements in 3D view

### PR DESCRIPTION
**Pull request by mkow**

### CODE REVIEW

File: src/visualization/manipulator.cc [hunk 1]
```diff
@@ -288,7 +290,6 @@ bool FreeManipulator::mouseEvent(QObject* watched, QMouseEvent* event) {
         }
         prev_pos_ = event->pos();
       }
-      return true;
```
Comment (altran01) [inline src/visualization/manipulator.cc]: Why those changes to don't filter out those handled mouse events (we still filter out keyboard/wheel though)?

File: src/visualization/trigram.cc [hunk 2]
```diff
@@ -305,16 +312,15 @@ void TrigramWidget::setManipulator(Manipulator* manipulator) {
 
 void TrigramWidget::autoSetBrightness() {
   auto new_brightness = suggestBrightness();
-  if (new_brightness == brightness_) return;
-  brightness_ = new_brightness;
-  if (brightness_slider_ != nullptr) {
```
Comment (altran01) [inline src/visualization/trigram.cc]: Eh, I'm not a great fan of removing braces (but not a big deal either way).

File: src/visualization/manipulator.cc [hunk 3]
```diff
@@ -100,14 +101,13 @@ QString TrackballManipulator::manipulatorName() {
 
 bool TrackballManipulator::eventFilter(QObject* watched, QEvent* event) {
   QWidget* widget = dynamic_cast<QWidget*>(watched);
-  if (widget == nullptr) {
```
Comment (adamiwaniuk) [inline src/visualization/manipulator.cc]: why you changed that?

File: src/visualization/manipulator.cc (hunk 1 above)
Comment (mkow) [inline src/visualization/manipulator.cc]: As discussed privately: it doesn't matter in this case, but we may change this in the future.

File: src/visualization/trigram.cc (hunk 2 above)
Comment (mkow) [inline src/visualization/trigram.cc]: I prefer to not have them in such shorts and simple ifs (IMO: no gain in readability if indentation is correct, but one line longer). I probably won't touch them in future PRs ;)

File: src/visualization/manipulator.cc (hunk 3 above)
Comment (mkow) [inline src/visualization/manipulator.cc]: I thought that Google style guide prefers `!widget`, but seems that's true only for Python style guide. Keep it or revert?

**Comment by mkow** [comment]
@altran01: seems that initial lag was there before this commit, but was harder to notice because of the glitches.

**Pull request merged**